azure-pipelines.yml

```diff
@@ -74,4 +74,5 @@ steps:
     displayName: 'run danger on PRs'
     condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'))
     env:
+      DANGER_DISABLE_TRANSPILATION: true
```

Comment:
This is done to get around

> Error: Reentrant plugin detected trying to load "/home/vsts/work/1/s/node_modules/babel-plugin-transform-destructuring-fork/src/index.js". This module is not ignored and is trying to load itself while compiling itself, leading to a dependency cycle. We recommend adding it to your "ignore" list in your babelrc, or to a .babelignore.

That being said, with us using Node.js 10, I don't think we need to transpile Dangerfile.